src/tests/Details.test.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import {render} from '@testing-library/react'
+import Details from '../pages/Details'
+
+
+describe('details', () => {
+    test('renders the details component', () => {
+      render(<Details />);
```

Comment:
Can you assert something? maybe you can check that element with role='heading' exist in the page